react-16/src/CatchTheRedGame.js

```diff
@@ -0,0 +1,60 @@
+import React, {useState} from 'react';
+
+const Board = (props) => {
+  const {setScore, redIdx, setRedIdx} = props;
+
+  function handleClick(e, idx) {
+    if (idx === redIdx) {
+      setScore(score => score + 10);
+      setRedIdx(Math.floor(Math.random() * (10)));
+    } else {
+      setScore(score => score - 5);
+    }
+  }
+
+  return (
+      <div>
+        {[...Array(10).keys()].map((o, i) =>
+            <div
+                style={{
```

Comment:
In general, don't use inline classes unless they are modified by React.
Use either a variable or a CSS file or JSS/Styled components.